bookkeeper-server/src/main/java/org/apache/bookkeeper/server/http/service/ListUnderReplicatedLedgerService.java

```diff
@@ -51,12 +49,12 @@
     static final Logger LOG = LoggerFactory.getLogger(ListUnderReplicatedLedgerService.class);
 
     protected ServerConfiguration conf;
-    protected ZooKeeper zk;
+    protected LayoutManager layoutManager;
 
-    public ListUnderReplicatedLedgerService(ServerConfiguration conf, ZooKeeper zk) {
+    public ListUnderReplicatedLedgerService(ServerConfiguration conf, LayoutManager layoutManager) {
```

Comment:
Take LedgerManagerFactory from AutoRecovery

Reply:
Thanks, will change it.